lib/AFS/CellCC/DB.pm

```diff
@@ -69,19 +69,35 @@ _connect($$$$) {
 # Connect to the db with a read/write connection
 sub
 connect_rw() {
+    my %opts;
+    my %timeout = %{ config_get('db/rw/timeout') };
+
+    while (my ($key, $value) = each %timeout) {
+	$opts{$key} = $value;
```

Comment:
okay, so this is setting any arbitrary dbi option. that's useful, but then it shouldn't be called "timeout", since it's used for anything. just calling it "options" I think sounds fine?

Reply:
Ack.